yew/src/virtual_dom/vnode.rs

```diff
@@ -32,6 +39,22 @@ pub enum VNode {
     VRef(Node),
 }
 
+#[cfg(feature = "ssr")]
+impl TryFrom<VNode> for Html {
+    type Error = HtmlStringifyError;
+
+    fn try_from(value: VNode) -> Result<Html, HtmlStringifyError> {
+        let html = match value {
```

Comment:
I think this could be done without the `let` binding.
```suggestion
        Ok(match value {
```